.github/workflows/go.yml

```diff
@@ -62,4 +62,6 @@ jobs:
         run: go test -race -coverprofile=coverage.txt -covermode=atomic ./...
 
       - name: Upload coverage to Codecov
-        run: bash <(curl -s https://codecov.io/bash)
+        uses: codecov/codecov-action@v2
+        with:
+          files: ./coverage.txt
```

Comment:
This file should be autodetected by the action. Do we really need to be explicit here?

Reply:
Not really :new_moon_with_face: I'll remove it :+1: